templates/helpers/_field_blocks.twig

```diff
@@ -43,11 +43,14 @@
 {% block extended_fields %}
 
     {# Special case for 'select' fields: if it's a multiple select, the field is an array. #}
+    {# I dont know what is going on here, but field.selectedIds for pgsql showcase/mr-jean-feeney (and others) #}
+    {# yield field values, NOT integer IDs.. Also, selectedIds does not exists (try dump(field) ) #}
+    {# TODO: FIX IT, bypassing for now by setting field.id #}	
```

Comment:
This is weird. Might be an unrelated bug, that's coming to light here.. I'll check this one out as well.

Reply:
Ok, took a look at this. Not sure why it was failing on your end.. `{{ field.id }}` is not a good fix, because it's an entirely different ID than the one(s) we need. 

It uses method `getSelectedIds` from `src/Entity/Field/SelectField.php`, so it really should exist, if the field is a SelectField.

One thing you might try is to explicitly set it to return an _array_ instead of one item: 

```
{% setcontent selected = field.contentType where {'id': field.selectedIds} returnmultiple %}
```

Reply:
Should be fixed now. selectedIds function returned full (string) values, instead of (integer)IDs. So, why not select on field.value: .. instead of field.id: .. :) 
Works nicely in postgres, I still have to run a test for the others.